mlflow/utils/proto_json_utils.py

```diff
@@ -103,9 +104,8 @@ def _merge_json_dicts(from_dict, to_dict):
     return to_dict
 
 
-def message_to_json(message):
-    """Converts a message to JSON, using snake_case for field names."""
-
+def message_to_dict(message):
```

Comment:
Can we add type hints to these methods while we're here?

Reply:
ok

Reply:
added!